Port 9151 and Goerli ETH in Ursula guide (#1415)
nucypher/nucypher: docs/source/guides/ursula_configuration_guide.rst

You can also view your node’s network status webpage by navigating your web browser to ``https://<your-node-ip-address>:9151/status``.

> It's a good idea to check that this URL opens from the outside: it means that

Reviewer: ```suggestion
It's a good idea to ensure that this URL can be accessed publicly: it means that
```

> your node can be seen by other NuCypher nodes also.

Reviewer 2: ```suggestion
your node can be seen by other NuCypher nodes.
```
